sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Fixed
 
 - Fixed concurrency issue in our challenge-based authentication policy ([#9737](https://github.com/Azure/azure-sdk-for-net/issues/9737))
+- Fixed The return value of issuer name always be null ([#10908](https://github.com/Azure/azure-sdk-for-net/issues/10908))
+- Fixed Method GetIssuerAsync() throw a exception while try to get issuer pre created by portal ([#10905](https://github.com/Azure/azure-sdk-for-net/issues/10905))
```

Comment:
Same here, e.g. "Fixed an issue that threw an exception if an issuer was created in the portal". The customer can navigate to the bug for more info, but this sentence doesn't make sense otherwise.